### PR TITLE
GH-2398 - Adding-missing-EIP-3770-prefixes 

### PR DESCRIPTION
**Pull request by biafra23**
Handles #2398 

Changes proposed in this pull request:
- Add missing EIP-3770 prefixes to added and removed owner addresses

Add / remove | Add | Remove
------------- | ---- | --------
image | image | image

### CODE REVIEW

File: Multisig/UI/Settings/OwnerKeyManagement/AddOwnerKey/EnterOwnerAddressViewController.swift [hunk 1]
```diff
@@ -66,7 +66,7 @@ class EnterOwnerAddressViewController: UIViewController {
                 info: nil,
                 chainId: self.safe.chain!.id!)
 
-            self.addressField.setAddress(address, label: resolvedName)
+            self.addressField.setAddress(address, label: resolvedName, prefix: self.safe.chain!.shortName)
```
Comment (MouazAlzahabi) [inline Multisig/UI/Settings/OwnerKeyManagement/AddOwnerKey/EnterOwnerAddressViewController.swift]: For safer code we can remove the force unwrapping so we use 
```suggestion
            self.addressField.setAddress(address, label: resolvedName, prefix: self.safe.chain?.shortName)
```

**Review by MouazAlzahabi — APPROVED** [review]

File: Multisig/UI/Settings/OwnerKeyManagement/AddKeyAsNewOwner/ReviewAddOwnerTxViewController.swift [hunk 2]
```diff
@@ -75,7 +75,7 @@ class ReviewAddOwnerTxViewController: ReviewSafeTransactionViewController {
                                                     info: nil,
                                                     chainId: safe.chain!.id!)
 
-        cell.set(owner: AddressInfo(address: owner, name: name), action: .addingOwner)
+        cell.set(owner: AddressInfo(address: owner, name: name), action: .addingOwner, prefix: safe.chain!.shortName)
```
Comment (MouazAlzahabi) [inline Multisig/UI/Settings/OwnerKeyManagement/AddKeyAsNewOwner/ReviewAddOwnerTxViewController.swift]: For safer code we can remove the force unwrapping so we use 

```suggestion
            self.addressField.setAddress(address, label: resolvedName, prefix: self.safe.chain?.shortName)
```

**Review by DmitryBespalov — CHANGES_REQUESTED** [review]
please address @MouazAlzahabi 's comments :pray:

File: Multisig/UI/Settings/OwnerKeyManagement/AddOwnerKey/EnterOwnerAddressViewController.swift (hunk 1 above)
Comment (biafra23) [reply Multisig/UI/Settings/OwnerKeyManagement/AddOwnerKey/EnterOwnerAddressViewController.swift]: Good catch! Thanks!

**Review by DmitryBespalov — APPROVED** [review]

**Pull request merged**